Packs/CiscoWSA/Integrations/CiscoWSAV2/CiscoWSAV2.py

```diff
@@ -0,0 +1,1526 @@
+    policy_description = args.get("policy_description")
+    policy_expiry = args.get("policy_expiry")
+
+    response = client.access_policy_create_request(
+        policy_name=policy_name,
+        policy_status=policy_status,
+        policy_order=policy_order,
+        identification_profile_name=identification_profile_name,
+        policy_description=policy_description,
+        policy_expiry=policy_expiry,
+    )
+
+    if response.status_code == 204:
+        return CommandResults(
+            readable_output=f'Created "{policy_name}" access policy successfully.'
+        )
+    else:
+        raise DemistoException(response.json())
+
+
+def access_policy_update_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    """
+    Update an access policy.
+
+    Args:
+        client (Client): Cisco WSA API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: readable outputs for XSOAR.
+    """
+    policy_name = args["policy_name"]
+    new_policy_name = args.get("new_policy_name")
+    policy_status = args.get("policy_status")
+    policy_description = args.get("policy_description")
+    policy_order = arg_to_number(args.get("policy_order"))
+    policy_expiry = args.get("policy_expiry")
+
+    response = client.access_policy_update_request(
+        policy_name=policy_name,
+        new_policy_name=new_policy_name,
+        policy_status=policy_status,
+        policy_description=policy_description,
+        policy_order=policy_order,
+        policy_expiry=policy_expiry,
+    )
+
+    if response.status_code == 204:
+        readable_output = f'Updated "{policy_name}" access policy successfully.'
+    else:
+        raise DemistoException(response.json())
+
+    return CommandResults(
+        readable_output=readable_output,
+    )
+
+
+def access_policy_protocols_user_agents_update_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    """
+    Update access policy's protocols and user agents settings.
+
+    Args:
+        client (Client): Cisco WSA API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: readable outputs for XSOAR.
+    """
+    policy_name = args["policy_name"]
+    block_custom_user_agents = argToList(args.get("block_custom_user_agents"))
+    allow_connect_ports = argToList(args.get("allow_connect_ports"))
+    block_protocols = argToList(args.get("block_protocols"))
+
+    response = client.access_policy_protocols_user_agents_update_request(
+        policy_name=policy_name,
+        block_custom_user_agents=block_custom_user_agents,
+        allow_connect_ports=allow_connect_ports,
+        block_protocols=block_protocols,
+    )
+
+    if response.status_code == 204:
+        readable_output = f'"{policy_name}" access policy updated successfully.'
+    else:
+        raise DemistoException(response.json())
```

Comment:
I see that this code returns a lot.
You should define an error handler command to handle all of these errors.
What you should do is to override the _http_request command and call it with error_handler = YOUR_ERROR_HANDLER.
Finaly you will be able to remove the try/catch :)
I implemented it in Fortiweb.